bigquery/google/cloud/bigquery/query.py

```diff
@@ -31,7 +31,6 @@ class _SyncQueryConfiguration(object):
     Values which are ``None`` -> server defaults.
     """
     _default_dataset = None
-    _dry_run = None
```

Comment:
So there's actually no property `dryRun` on the Query object.

I looked through the docs you linked pretty hard before I figured this out.

Here's a screenshot of the JSON from that doc page, but showing `dryRun` at the `configuration` level.

<img width="349" alt="screen shot 2017-02-06 at 12 02 18 pm" src="https://cloud.githubusercontent.com/assets/192456/22657384/4d5687f0-ec64-11e6-83b6-343caf132b31.png">


I could be mistake, I'm not super familiar with BigQuery but I can't find anything about this option being on a specific Query.